sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -100,6 +110,51 @@ export interface CreateConnectionContextBaseParameters {
   operationTimeoutInMs?: number;
 }
 
+class CoreAmqpConnection extends Connection {
+  /**
+   * Creates an amqp sender link. Max listener limit on the sender is set to 1000 because the
+   * default value of 10 in NodeJS is too low.
+   * @param options - Optional parameters to create a sender link.
+   * @returns Promise<Sender>.
+   */
+  async createSender(options?: CreateSenderOptions): Promise<Sender> {
+    const sender = await super.createSender(options);
+    sender.setMaxListeners(1000);
```

Comment:
should this `1000` be put in a variable instead so it is easier to update it everywhere if there is a need to?

Reply:
Good point!

Reply:
extracted